GUI/GUIMod.cs

```diff
@@ -115,7 +115,27 @@ public GUIMod(CkanModule mod, IRegistryQuerier registry, KspVersionCriteria curr
             // KSP.
             if (latest_available_for_any_ksp != null)
             {
-                KSPCompatibility = KSPCompatibilityLong = registry.LatestCompatibleKSP(mod.identifier)?.ToString() ?? "any";
+                string str = registry.LatestCompatibleKSP(mod.identifier)?.ToString();
+                int major = registry.LatestCompatibleKSP(mod.identifier).Major;
```

Comment:
Please don't call `LatestCompatibleKSP` more than once. It loops over all versions of the mod checking their compatibility, so the return value should go into a local variable.